src/hooks/Rules.py

```diff
@@ -28,39 +28,47 @@ def requiresMelee(world: World, multiworld: MultiWorld, state: CollectionState,
     """Returns a requires string that checks if the player has unlocked the tank."""
     return "|Figher Level:15| or |Black Belt Level:15| or |Thief Level:15|"
 
-def ItemValue(world: World, multiworld: MultiWorld, state: CollectionState, player: int, args: str):
+def ItemValue(world: World, multiworld: MultiWorld, state: CollectionState, player: int, valueCount: str, noCache: Optional[str] = None):
     """When passed a string with this format: 'valueName:int',
     this function will check if the player has collect at least 'int' valueName worth of items\n
-    eg. {ItemValue(Coins:12)} will check if the player has collect at least 12 coins worth of items
+    eg. {ItemValue(Coins:12)} will check if the player has collect at least 12 coins worth of items\n
+    You can add a second string argument to disable creating/checking the cache like this:
+    '{ItemValue(Coins:12,Disable)}' it can be any string you want
```

Comment:
The two places that use `noCache` below use it as a bool. So we'd either want it to be bool in the type hint above (for true/false) or int (for 0/1). Whichever seems most convenient for people building their worlds. 😄 

The way it is now, I could put "enabled" or "on" or "always_cache", and all of those would disable the cache, haha.

Reply:
That makes sense, yeah. I think the goal should be to remove ambiguity as much as possible. Non-empty string always evaluating to True is less intuitive than either (1) converting string True/False/true/false in func args to bool or (2) using 0 and 1 and doing a `bool()` call. Just unfortunately something that wasn't considered with req functions because every arg is a string by default.

Reply:
I could write some code to identify the type of a require fonction args and then try and convert the user's input to said type and probably raise exception if the conversion I program fail? that might be something for another pr

Reply:
I got some argument conversion code ready I just dont know if I add it to this pr or if we make another